pkg/featuregate/featuregate.go

```diff
@@ -50,6 +50,11 @@ var (
 		featuregate.StageBeta,
 		featuregate.WithRegisterDescription("controls whether the operator supports Apache HTTPD auto-instrumentation"))
 
+	EnableMultiInstrumentationSupport = featuregate.GlobalRegistry().MustRegister(
+		"operator.autoinstrumentation.multi-instrumentation",
+		featuregate.StageAlpha,
+		featuregate.WithRegisterDescription("controls whether the operator supports multi instrumentation"))
```

Comment:
Please add a `featuregate.FromVersion` option to the feature gate.

Reply:
Should it be `featuregate.WithRegisterFromVersion(VERSION)`? Where `VERSION` is next unreleased so `0.82.0`?